entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
We should set pipefail in our bash scripts:`set -euf -o pipefail`

([ref](https://sipb.mit.edu/doc/safe-shell/))

Reply:
as a general rule of thimb I guess this is true, but in this case:
- `pipefail` is not portable
- `-u` will break checks for optional arguments (which may be legitimately unset)
- `-e` will break checking for non-zero exit codes (like grep didn't find or test eval to false)

maybe there's a way to rewrite the entire script to work in strict mode but I'm not sure it's worth the effort